Bug 1887361: Move manifest to 4.7

jcantrill opened the pull request:
This PR bumps the manifest to 4.7 and replaces https://github.com/openshift/cluster-logging-operator/pull/736


ref: https://bugzilla.redhat.com/show_bug.cgi?id=1887361

Fixes #742

periklis commented on manifests/cluster-logging.package.yaml:
```diff
@@ -1,5 +1,5 @@
+#! package-manifest: ./deploy/chart/catalog_resources/rh-operators/clusterlogging.v4.7.0.clusterserviceversion.yaml
 packageName: cluster-logging
 channels:
 - name: "4.6"
```
Shouldn't the channel name be also `4.7`?

periklis commented on Dockerfile.e2e:
```diff
@@ -2,7 +2,7 @@
 # test scripts to deploy the elasticsearch operator and then
 # test the clusterlogging operator
-FROM registry.svc.ci.openshift.org/ocp/4.6:elasticsearch-operator-src
+FROM registry.svc.ci.openshift.org/ocp/4.7:elasticsearch-operator-src
```
FYI, this is actually the blocker that requires us to merge this manually.

jcantrill replied:
reverted to 4.6 for now... should matter for CLO

periklis commented:
Why does this PR delete:

jcantrill commented:
> Why does this PR delete:
> ![image](https://user-images.githubusercontent.com/152312/96882491-a6fe0700-147f-11eb-8262-1ebc6a40619c.png)

@vimalk78 do you know why?  i just rebased and fixed up your PR

vimalk78 commented:
> Why does this PR delete:
> ![image](https://user-images.githubusercontent.com/152312/96882491-a6fe0700-147f-11eb-8262-1ebc6a40619c.png)



> > Why does this PR delete:
> > ![image](https://user-images.githubusercontent.com/152312/96882491-a6fe0700-147f-11eb-8262-1ebc6a40619c.png)
> 
> @vimalk78 do you know why? i just rebased and fixed up your PR

these files are not deleted, the bundle is generated again after changing the manifest folder in root of repo.

periklis reviewed (approved):
/lgtm

ewolinetz commented:
/hold
please adjust the skip range to be 4.5 -> 4.7

vimalk78 commented:
/retest

ewolinetz commented:
/hold cancel
/lgtm
